src/ucp/api/ucp.h

```diff
@@ -2481,15 +2481,27 @@ ucp_atomic_fetch_nb(ucp_ep_h ep, ucp_atomic_fetch_op_t opcode,
  *
  * @param [in]  request     Non-blocking request to check.
  *
- * @param [out] info        If request is in completed state, it is
- *                          filled with the details about the message.
+ * @return Error code as defined by @ref ucs_status_t
+ */
+ucs_status_t ucp_request_check_status(void *request);
+
+
+/**
+ * @ingroup UCP_COMM
+ * @brief Check the status and currently available state of non-blocking request
+ *        returned from @ref ucp_tag_recv_nb routine.
  *
- * @note The @p info parameter is relevant for receive operations only. It is
- * left uninitialized in case of send operation.
+ * This routine checks the state and returns current status of the request
+ *      returned from @ref ucp_tag_recv_nb routine. Any value different from
```

Comment:
s/ucp_tag_recv_nb/ucp_tag_recv_nbr/

Reply:
Thnks! Not exactly. I'll add ucp_tag_recv_nbr to description as well.

Reply:
Ahh...yes, I overlooked that ucp_tag_recv_nb always returns a request handle.